selftests.sh

```diff
@@ -6,22 +6,52 @@ function run_device_selftest()
     echo $?
 }
 
-exit_code=0
-for DEVICE_TYPE in ./devices/*
-do
-    for TEST in ./testcases/*.yaml
+function test_for_android_templates(){
+    local exit_code=0
+    for F_DEVICE_TYPE in ./devices/android/*; do
+        DEVICE_TYPE=$(basename "${F_DEVICE_TYPE}")
+        for F_TESTCASE in ./testcases/android/*.yaml; do
+            TEST=$(basename ${F_TESTCASE})
+            ./submit_for_testing.py \
+                --testplan-path ./testcases/android/ \
+                --testplan-device-path ./devices/android/ \
+                --test-case ${TEST} \
+                --variables variables-android.ini \
+                --device-type ${DEVICE_TYPE} \
+                --dry-run \
```

Comment:
Don't need a \ at the end

Reply:
removed it now to avoid confusion. I left it there because I often forget to add the backslash when I added another new parameter to the command by copying the last line.